pkg/compute/kube/port_forward.go

```diff
@@ -0,0 +1,23 @@
+package kube
+
+import (
+	"fmt"
+
+	"k8s.io/client-go/rest"
+
+	"github.com/openservicemesh/osm/pkg/models"
+)
+
+// EnvoyConfigFromProxy takes the given proxy, port forwards to the pod from this proxy, and returns the envoy config
+func (c *client) EnvoyConfigFromProxy(proxy *models.Proxy, url string, kubeConfig *rest.Config) (string, error) {
```

Comment:
nit: rename to `ConfigFromProxy`, change `url` to `configType`, and move this to `client.go`